development/compiling/compiling_with_mono.rst

```diff
@@ -48,6 +48,23 @@ Notes
 -  **Do not** build normal binaries with ``mono_glue=no``. This option disables C# scripting and therefore must only be used for the temporary binary that will be used to generate the glue. Godot should print a message at startup warning you about this.
 -  The glue sources must be regenerated every time the ClassDB API changes. If there is an API mismatch with the generated glue, Godot will print an error at startup.
 
+Example (Windows) (with -j6 boost)
+-------------
+
+::
+
+    # Build temporary binary
+    scons -j6 p=windows tools=yes module_mono_enabled=yes mono_glue=no
+    # Generate glue sources
+    bin\godot.windows.tools.64.mono --generate-mono-glue modules/mono/glue
+
+    ### Build binaries normally
+    # Editor
+    scons -j6 p=windows target=release_debug tools=yes module_mono_enabled=yes
+    # Export templates
+    scons -j6 p=windows target=debug tools=no module_mono_enabled=yes
+    scons -j6 p=windows target=release tools=no module_mono_enabled=yes
```

Comment:
I think there is no point in including the -j6 argument in this example.

Reply:
Ok